surefire-its/src/test/java/org/apache/maven/surefire/its/JUnitPlatformEnginesIT.java

```diff
@@ -63,7 +63,7 @@
         args.add( new Object[] { "1.1.1", "5.1.1", "1.0.0", "1.0.0" } );
         args.add( new Object[] { "1.2.0", "5.2.0", "1.1.0", "1.0.0" } );
         args.add( new Object[] { "1.3.1", "5.3.1", "1.1.1", "1.0.0" } );
-        args.add( new Object[] { "1.4.0-SNAPSHOT", "5.4.0-SNAPSHOT", "1.1.1", "1.0.0" } );
+        // TODO args.add( new Object[] { "1.4.0-SNAPSHOT", "5.4.0-SNAPSHOT", "1.1.1", "1.0.0" } );
```

Comment:
This should be solved before merging. I guess, the Surefire-internal resolver tool needs a version update to support the transport protocols offered by https://oss.sonatype.org/content/repositories/snapshots and others.

Reply:
Perhaps, we should move this "Resolve missing artifact" feature to 3.0 then?

Reply:
In that case the 3.0 might be a good candidate. Please do not commit to branch 3.0.0-M1 because it is only about API 3.0 and it is still not finished. There are only details missing. We should create another branch for JUnit5 and check it out, especially the artifact resolution or check it out in your local working copy.

Reply:
Rebased the branch onto `master`.

Reply:
...just prepared to get this PR ready for being merged into a potential 3.0.0-M2.